apps/system/test/unit/app_chrome_test.js

```diff
@@ -464,11 +464,23 @@ suite('system/AppChrome', function() {
   });
 
   suite('URLBar', function() {
-    test('click', function() {
+    test('click when collapsed', function() {
```

Comment:
Thanks for getting to this so quickly Alberto! I think the global-search-request event dispatcher is covered by line 482? It's also covered by the integration test.